layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java

```diff
@@ -15,37 +15,44 @@
  */
 package org.moditect.layrry.launcher.internal;
 
+import com.beust.jcommander.Parameter;
+
 import java.io.File;
 import java.util.ArrayList;
 import java.util.List;
 
-import com.beust.jcommander.Parameter;
-
 public class Args {
 
     @Parameter(description = "Main arguments")
     private List<String> mainArgs = new ArrayList<>();
 
     @Parameter(names = "--layers-config", required = true, description = "Layers configuration file")
-    private File layersConfig;
+    private String layersConfig;
 
     @Parameter(names = "--properties", description = "Additional config properties")
     private File properties;
 
+    @Parameter(names = "--basedir", description = "Base directory for resolving relative paths")
```

Comment:
Could this be optional (defaulting to the current directory, from which Layrry was launched)?

Reply:
I believe is already treated as optional if you look at `LayrryLauncher`. If no value is given:
- if layers file is remote then `System.getProperty("user.dir")` is used.
- if layers file is local the `layesFilePath.parent()` is used, which is the current behavior before this PR.